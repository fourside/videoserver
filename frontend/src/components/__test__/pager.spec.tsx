import * as React from 'react';
import 'jest';
import * as renderer from 'react-test-renderer';

import Pager from '../pager';

test('snapshot', () => {
  const component = renderer
    .create(<Pager total={45} currentPage={2} onChange={() => {}} />)
    .toJSON();
  expect(component).toMatchSnapshot();
});

