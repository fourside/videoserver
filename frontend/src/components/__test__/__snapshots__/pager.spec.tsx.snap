// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<nav
  aria-label="pagination"
  className="pagination is-small sc-bdVaJa ZacCn"
  role="navigation"
>
  <ul
    className="pagination-list"
  >
    <li>
      <a
        aria-label="Goto page 0"
        className="pagination-link"
        onClick={[Function]}
      >
        0
      </a>
    </li>
    <li>
      <a
        aria-label="Goto page 1"
        className="pagination-link"
        onClick={[Function]}
      >
        1
      </a>
    </li>
    <li
      aria-current="page"
      aria-label="Page 2"
      className="pagination-link is-current"
    >
      2
    </li>
  </ul>
</nav>
`;
